lib/ansible/plugins/callback/default.py

```diff
@@ -293,11 +293,12 @@ def v2_runner_item_on_skipped(self, result: CallbackTaskResult) -> None:
             self._display.display(msg, color=C.COLOR_SKIP)
 
     def v2_playbook_on_include(self, included_file):
-        msg = 'included: %s for %s' % (included_file._filename, ", ".join([h.name for h in included_file._hosts]))
-        label = self._get_item_label(included_file._vars)
-        if label:
-            msg += " => (item=%s)" % label
-        self._display.display(msg, color=C.COLOR_INCLUDED)
+        if self.get_option("display_included_hosts"):
```

Comment:
If this is accepted, it's better to invert this check, reducing the unnecessary indentation throughout the function.